<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XDiskGUIClass</class>
 <widget class="QWidget" name="XDiskGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XDiskGUI</string>
  </property>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="filelistWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>611</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文件大小</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="pathEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>./server_root</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="portBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>171</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>21002</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>99</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器 IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>139</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器 端口</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xdisk_gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>XDiskGUIClass</receiver>
   <slot>Refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadButton</sender>
   <signal>clicked()</signal>
   <receiver>XDiskGUIClass</receiver>
   <slot>Upload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Refresh()</slot>
  <slot>Upload()</slot>
 </slots>
</ui>
